plexapi/myplex.py

```diff
@@ -874,6 +889,24 @@ def link(self, pin):
         data = {'code': pin}
         self.query(self.LINK, self._session.put, headers=headers, data=data)
 
+    def _toOnlineMetadata(self, objs):
+        """ Convert a list of media objects to online metadata objects. """
+        # TODO: Add proper support for metadata.provider.plex.tv
+        # Temporary workaround to allow reloading and browsing of online media objects
+        if not isinstance(objs, list):
+            objs = [objs]
+        for obj in objs:
+            obj._server = PlexServer(self.METADATA, self._token)
```

Comment:
A request is sent to METADATA for each item of the watchlist. It seems to be problematic for huge watchlist because plex server can't handle so many requests.

https://github.com/Taxel/PlexTraktSync/issues/1048